server/routes/settings/radarr.ts

```diff
@@ -46,7 +46,13 @@ radarrRoutes.post<
       url: RadarrAPI.buildUrl(req.body, '/api/v3'),
     });
 
-    const { urlBase } = await radarr.getSystemStatus();
+    const urlBase =
+      req.body.baseUrl && req.body.baseUrl !== '/'
+        ? req.body.baseUrl
+        : await radarr
+            .getSystemStatus()
+            .then((value) => value.urlBase)
+            .catch(() => undefined);
```

Comment:
Why not always fetch the base URL, and just fall back to the user-inputted value if an exception is thrown?

Also, any reason why we don't want to apply this same logic to Sonarr?

Reply:
I fetch the base URL only if there's no base URL that the user provided already to be slightly faster.

As for Sonarr, I just missed it when I was making this fix lol 😅 I'll add that

Reply:
What if the user inputted an incorrect base URL?  Would be nice to try to fetch the correct one.  I don't imagine users are editing their *arr servers all the time, so I don't see the harm in always attempting the fetch.

Reply:
We could do something like this?
```suggestion
    const { urlBase } = await radarr
      .getSystemStatus()
      .then((value) => value.urlBase)
      .catch(() => req.body.baseUrl);
```

Reply:
That's good 😄 
I just don't personally like automatically fixing data that a user has already input. But here it should be no problem